project/src/com/google/daggerqueryui/scripts/binding_graph.js

```diff
@@ -138,6 +167,50 @@ const bindingGraph = (function() {
     }).length !== 0;
   }
 
+  /**
+   * Supports click and double click events in the network object.
+   * @param {vis.Network} network
+   */
+  function supportEventsRecognition(network) {
```

Comment:
Are the two events currently:
1) Click twice with the left button for deps 
2) Click once with the right button for ancestors

If so, can we update each behaviour to be the same # of clicks or the same button? eg. 
1) Click once with the left button for deps 
2) Click once with the right button for ancestors

Reply:
Yes, two events are exactly as you described now. At the beginning I though about using left-click and `double-click`, but it looks like `double-click` always raises two `left-click`. In vis.js docs they recommend to fix it by measuring time between clicks, but I don't like this approach since it is not really stable.

Instead, let's use `right-click`. I removed context menu [in this PR](https://github.com/googleinterns/dagger-query/pull/48) and it is easy to use it.

Now a single left-click action selects the node and its edges, but I don't think it really matters, so I don't see any problems using a single-click action instead of a double-click.

Reply:
[Done](https://github.com/googleinterns/dagger-query/pull/47/commits/c6ffff60fb90ebae0f58b45c39b750977191b6c7)!